quarkus-workshop-super-heroes/super-heroes/rest-villain/src/test/java/io/quarkus/workshop/superheroes/villain/VillainResourceTest.java

```diff
@@ -62,15 +62,15 @@ void shouldPingSwaggerUI() {
     @Test
     void shouldPingLiveness() {
         given()
-            .when().get("/health/live")
+            .when().get("/q/health/live")
```

Comment:
That should be done also for `hero` and `fight` microservices

Reply:
This needs also be reported to the html pages...

Reply:
Done